Kitodo/src/test/java/test/TreeDeleter.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package test;
+
+import java.io.IOException;
+import java.nio.file.FileVisitResult;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.SimpleFileVisitor;
+import java.nio.file.attribute.BasicFileAttributes;
+import static java.nio.file.FileVisitResult.*;
```

Comment:
```suggestion
```
Since you already import `java.nio.file.FileVisitResult` in line 15, it is not required statically import the individual constants from the class (and if you did please do not use wildcard imports, as our coding guidelines indicate that we should use individual imports)